src/vectorclock.erl

```diff
@@ -130,6 +104,15 @@ set_clock_of_dc(DcId, Time, VectorClock) ->
 from_list(List) ->
     dict:from_list(List).
 
+max([]) -> dict:new();
```

Comment:
I talked with Alejandro, I thought he referred to the min/max functions, but he meant the whole branch. So never mind.